sites/main-page-title/includes/main-page-title.html

```diff
@@ -1,7 +1,12 @@
-{%- if page.sectionTitle -%}
+{%- if page.sectionTitle or page.byline -%}
 <hgroup id="wb-cont">
+	{%- if page.sectionTitle -%}
 	<p>{{ page.sectionTitle }}</p>
+	{%- endif -%}
 	<h1 property="name">{{ page.title }}</h1>
+	{%- if page.byline -%}
+	<p>{% if page.language == "fr" %}De&nbsp;:{% else %}From:{% endif %} <a href="{{ page.byline.url }}">{{ page.byline.institution }}</a></p>
+	{%- endif -%}
```

Comment:
Do we make it default or should we make the author to enable this, like by using the gc-byline class on the last paragraph?

We just need to think long term, do we want to lock down the last paragraph for a byline that is now half demote and still alive on news pages knowing that there is an intention to get it replaced by `gc-contributor` instead. Let have an internal discussion about it. 

cc @Ricokola 

I do think we should have something like to allow us to re-use that last paragraph in the "hgroup" heading section for other purpose:

```suggestion
	{%- if page.byline -%}
	<p class="gc-byline">{% if page.language == "fr" %}De&nbsp;:{% else %}From:{% endif %} <a href="{{ page.byline.url }}">{{ page.byline.institution }}</a></p>
	{%- endif -%}
```

or 

```suggestion
<hgroup class="with-byline">
  ....
	{%- if page.byline -%}
	<p>{% if page.language == "fr" %}De&nbsp;:{% else %}From:{% endif %} <a href="{{ page.byline.url }}">{{ page.byline.institution }}</a></p>
	{%- endif -%}
</hgroup>
```